modules/ROOT/pages/questions/question-03.adoc

```diff
@@ -0,0 +1,16 @@
+3. https://ephocks.atlassian.net/browse/TINY-8333[TINY-8333] (removing
```

Comment:
@TheSpyder

Reply:
The plugin is functionally identical, so we shouldn't need to say anything other than it moved to a premium feature.